src/store/flow/utils/importHelpers.ts

```diff
@@ -49,6 +49,64 @@ export function mergeFlowContainer(
   return existingFlowContainerStore
 }
 
+interface CreateFlowStackBlock {
+  uuid: string;
+  type: string;
+  config: {
+    flow_id: string;
+  };
+}
+
+interface CreateFlowStackFlow {
+  uuid: string;
+  blocks: CreateFlowStackBlock[];
+}
+
+interface CreateFlowStackData {
+  flows: CreateFlowStackFlow[];
+}
+
+export function createContainerFlowStack(json_data: any, flow_stack: string[]): void {
+  const flows = json_data.flows;
+  const visitedFlows: string[] = [];
+
+  function process_flow(flow: CreateFlowStackFlow): void {
```

Comment:
in general, almost all functions we write are about processing something. This naming is not very explanatory.

- [ ] improve the naming to give more context about what it does

Reply:
Don't hesitate to leave a JS_doc if you think we need more longer context to help devs